docs/source/getting-started/registering-first-model/step3-load-model.rst

```diff
@@ -0,0 +1,84 @@
+Load a Registered Model
+=======================
+
+To perform inference on a registered model version, we need to load it into memory. There are many 
+ways to find our model version, but the best method differs depending on the information you have
+available. However, in the spirit of a quickstart, here is the method that requires the least amount
+of information.
```

Comment:
I think we should still add some explanation on what this code snippet is doing right? Just to provide more context to completely new users. It uses `mlflow.sklearn.load_model` to load a model from the model registry via a specific model URI format.

Reply:
Also, this method wouldn't work for non-sklearn model flavors. A more generic way to load the model version would be `mlflow.pyfunc.load_model`. Perhaps you could either swap it in, or add a note at the bottom explaining different flavors of loading models